pyproject.toml

```diff
@@ -22,18 +30,19 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Topic :: Software Development :: Quality Assurance",
 ]
+urls = { Documentation = "https://github.com/life4/deal" }
 
-[tool.flit.metadata.requires-extra]
+[project.optional-dependencies]
 all = [
-    "astroid>=2.11.0",    # for tyupe inference in linter
```

Comment:
This does not constrain the max version of astroid to be installed with deal. Both before and after the changes, your dependency manager should install the latest version of astroid. On the deal side, we only limit the min supported version because we know that the versions before that don't work with the current implementation.